website/docs/reference/dbt-jinja-functions/statement-blocks.md

```diff
@@ -3,6 +3,12 @@ title: "statement blocks"
 id: "statement-blocks"
 ---
 
+:::tip Recommendation
+
+The [run_query](/reference/dbt-jinja-functions/run_query) macro provides a more convenient way to run queries and fetch their results. It is a wrapper around statement blocks.
```

Comment:
```suggestion
We recommend using the `[run_query](/reference/dbt-jinja-functions/run_query)` macro instead of `statement` blocks. The `run_query` macro provides a more convenient way to run queries and fetch their results by wrapping `statement` blocks. This macro enables you to write more concise code that is easier to maintain.
```